### PR TITLE
i18n: add  ja/cn translations for ew raids

### PR DESCRIPTION
**Pull request by SJoshua**

### CODE REVIEW

File: ui/raidboss/data/06-ew/raid/p1s.ts [hunk 1]
```diff
@@ -89,6 +92,8 @@ const triggerSet: TriggerSet<Data> = {
           en: 'Close Shackles on YOU',
           de: 'Nahe Fesseln auf DIR',
           fr: 'Chaînes proches sur VOUS',
+          ja: '紫鎖（近い方）',
+          cn: '紫锁（近）',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p1s.ts:96]: suggest: 紫锁（近）点名

File: ui/raidboss/data/06-ew/raid/p1s.ts [hunk 2]
```diff
@@ -107,6 +112,8 @@ const triggerSet: TriggerSet<Data> = {
           en: 'Far Shackles on YOU',
           de: 'Entfernte Fesseln auf DIR',
           fr: 'Chaînes éloignées sur VOUS',
+          ja: '赤鎖（遠い方）',
+          cn: '红锁（远）',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p1s.ts:116]: suggest: 红锁（近）点名

File: ui/raidboss/data/06-ew/raid/p1s.ts [hunk 3]
```diff
@@ -309,11 +326,15 @@ const triggerSet: TriggerSet<Data> = {
           en: 'Shackles of Time on ${player}',
           de: 'Aspektierende Ketten auf ${player}',
           fr: 'Chaînes à retardement sur ${player}',
+          ja: '時限の魔鎖：${player}',
+          cn: '时限魔锁：${player}',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p1s.ts:330]: suggest: 时限魔锁点 ${player}

File: ui/raidboss/data/06-ew/raid/p1s.ts [hunk 4]
```diff
@@ -309,11 +326,15 @@ const triggerSet: TriggerSet<Data> = {
...
         },
         shacklesOnYou: {
           en: 'Shackles of Time on YOU',
           de: 'Aspektierende Ketten auf DIR',
           fr: 'Chaînes à retardement sur VOUS',
+          ja: '時限の魔鎖ついた',
+          cn: '时限魔锁：自己',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p1s.ts:337]: suggest: 时限魔锁点名

File: ui/raidboss/data/06-ew/raid/p2s.ts [hunk 5]
```diff
@@ -160,10 +164,14 @@ const triggerSet: TriggerSet<Data> = {
         arrowFirst: {
           en: 'Arrow First',
           de: 'Pfeil zuerst',
+          ja: '突進→散開',
+          cn: '箭头突进→散开',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p2s.ts:168]: suggest: 先突进 / 先对冲

File: ui/raidboss/data/06-ew/raid/p2s.ts [hunk 6]
```diff
@@ -213,6 +221,8 @@ const triggerSet: TriggerSet<Data> = {
         text: {
           en: 'Flare Tether',
           de: 'Flare Verbindung',
+          ja: 'フレアの線',
+          cn: '核爆的线',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p2s.ts:225]: suggest: 核爆连线

File: ui/raidboss/data/06-ew/raid/p2s.ts [hunk 7]
```diff
@@ -160,10 +164,14 @@ const triggerSet: TriggerSet<Data> = {
...
           en: 'Arrow First',
           de: 'Pfeil zuerst',
+          ja: '突進→散開',
+          cn: '箭头突进→散开',
         },
         spreadFirst: {
           en: 'Spread First',
           de: 'Verteilen zuerst',
+          ja: '散開→突進',
+          cn: '散开→箭头突进',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p2s.ts:174]: suggest: 先散开

**Comment by quisquous** [comment]
Hi! Thanks for the translations!  Would you mind fixing up the lint errors (`Warning:    86:14  warning  Expected locale object keys ordered like [en,de,fr,ja,cn,ko] ('ja' should be before 'ko')  rulesdir/cactbot-locale-order)` and doing a merge to fix the conflicts?

**Comment by SJoshua** [comment]
> Hi! Thanks for the translations! Would you mind fixing up the lint errors (`Warning: 86:14 warning Expected locale object keys ordered like [en,de,fr,ja,cn,ko] ('ja' should be before 'ko') rulesdir/cactbot-locale-order)` and doing a merge to fix the conflicts?

Sorry, it should be ok now.

**Comment by quisquous** [comment]
Can you also merge in main and fix these conflicts?

**Comment by SJoshua** [comment]
> Can you also merge in main and fix these conflicts?

Is this okay?

File: ui/raidboss/data/06-ew/raid/p1s.ts (hunk 2 above)
Comment (Crystallizednya) [reply ui/raidboss/data/06-ew/raid/p1s.ts]: 红锁（远）点名

**Pull request merged**